.travis.yml

```diff
@@ -57,22 +57,21 @@ jobs:
 
     - name: "Code imports"
       before_install: skip
-      install:
-        - pip install isort
+      install: pip install -r requirements/dev.txt
       before_script: skip
-      script: isort --recursive --check-only --diff .
+      # Set `ensure_newline_before_comments` to true, for compatibility with
```

Comment:
can't you specify this option in `setup.cfg` under the `isort` section?

Reply:
Good point, I changed it